corehq/ex-submodules/casexml/apps/case/templatetags/case_tags.py

```diff
@@ -226,6 +229,20 @@ def process_output(case_output, depth=0):
 
 
 def get_case_hierarchy(case, type_info):
+
+    def get_all_reverse_indices(case):
+        """
+        Return a list of child and extension type reverse indices for the given
+        case
+        """
+        children_indices = case.reverse_indices
```

Comment:
Just a thought. Should `case.reverse_indices` not include extension indexes by default?